### PR TITLE
Check config before deploy or rolling_update

### PR DESCRIPTION
**Pull request by liubo0127**
@LinuxGit @superlzs0476 PTAL

### CODE REVIEW

File: excessive_rolling_update.yml [hunk 1]
```diff
@@ -54,6 +54,13 @@
         - current_version.stdout_lines[0].replace(' ','').split(':')[1] < "v2.0.1"
         - tidb_version >= "v2.1.0" or tidb_version == "latest"
 
+- name: Pre-check for configuration
+  hosts: pd_servers[0]
+  tags:
+    - check_config
+  roles:
+    - check_config
```
Comment (zhouqiang-cl) [inline excessive_rolling_update.yml]: Did this run after deploy new binary

Comment (liubo0127) [reply excessive_rolling_update.yml]: This check run before deployment. If errors are checked after deployment, the configuration needs to be reupdated after modification

Comment (zhouqiang-cl) [reply excessive_rolling_update.yml]: maybe it have some error because the old version did not have the check command

Comment (zhouqiang-cl) [reply excessive_rolling_update.yml]: rest LGTM

Comment (liubo0127) [reply excessive_rolling_update.yml]: This may happen, we will reconsider.

Comment (liubo0127) [reply excessive_rolling_update.yml]: This does not happen, this check is to distribute the new binary and configuration files to a temporary directory to check

**Pull request merged**